src/FakeItEasy/A.cs

```diff
@@ -14,14 +14,14 @@ namespace FakeItEasy
     [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "A", Justification = "Is spelled correctly.")]
     public static class A
     {
-        private static IFakeConfigurationManager ConfigurationManager
+        internal static IFakeCreatorFacade FakeCreator

```

Comment:
> It could, but then I didn't want knowledge of how the service locator works (are they singletons or not?) to have to propagate to the new type. I thought it safest just to reuse.

Scratch that. It's a property which calls the service locator every time, so no reason `Sdk.Create` can't have the same property. I'll amend.